### PR TITLE
add Baidu Kunlun XPU Card support models for installation documents in Chinese/English

### PR DESCRIPTION
**Pull request by vslyu**
Add five models Baidu Kunlun XPU Card supported as following:
-  [ResNet50 ](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph/docs/zh_CN/extension/train_on_xpu.md) 
-  [MobileNetv3](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph/docs/zh_CN/extension/train_on_xpu.md) 
-  [Deeplabv3](https://github.com/PaddlePaddle/PaddleSeg/blob/develop/legacy/docs/train_on_xpu.md) 
-  [DQN](https://github.com/PaddlePaddle/PARL/blob/develop/examples/DQN/README.md) 
-  [Bertbase](https://github.com/PaddlePaddle/models/blob/develop/PaddleNLP/legacy/pretrain_language_models/BERT/README.md)

### CODE REVIEW

**Review by swtkiwi — APPROVED** [review]
LGTM

File: doc/paddle/install/install_Kunlun_zh.md [hunk 1]
```diff
@@ -182,7 +182,10 @@ o  ```python mnist_example.py --use_device=xpu --num_epochs=5```
...
 ## 附录：当前昆仑XPU适配支持的模型
 
-|  模型   | 下载链接  |
+|  模型名称  | 模型地址  |
 |  ----  | ----  |
-| ResNet50  | [点击下载](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/ResNet50_vd_pretrained.pdparams) |
-| MobileNetv3  | [点击下载](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/MobileNetV3_large_x1_0_pretrained.pdparams) |
+| ResNet50  | [模型地址](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph/docs/zh_CN/extension/train_on_xpu.md) |
+| MobileNetv3  | [模型地址](https://github.com/PaddlePaddle/PaddleClas/tree/dygraph/docs/zh_CN/extension/train_on_xpu.md) |
+| Deeplabv3  | [模型地址](https://github.com/PaddlePaddle/PaddleSeg/blob/develop/legacy/docs/usage.md) |
```
Comment (QingshuChen) [inline doc/paddle/install/install_Kunlun_zh.md]: 这个链接没有Deeplabv3的说明

File: doc/paddle/install/install_Kunlun_zh.md [hunk 2]
```diff
@@ -182,7 +182,10 @@ o  ```python mnist_example.py --use_device=xpu --num_epochs=5```
...
+| DQN  | [模型地址](https://github.com/PaddlePaddle/PARL/blob/develop/examples/DQN/README.md) |
```
Comment (QingshuChen) [inline doc/paddle/install/install_Kunlun_zh.md:190]: dqn也没有ready

File: doc/paddle/install/install_Kunlun_zh.md [hunk 3]
```diff
@@ -182,7 +182,10 @@ o  ```python mnist_example.py --use_device=xpu --num_epochs=5```
...
+| Bertbase  | [模型地址](https://github.com/PaddlePaddle/models/blob/develop/PaddleNLP/legacy/pretrain_language_models/BERT/README.md) |
```
Comment (QingshuChen) [inline doc/paddle/install/install_Kunlun_zh.md:191]: 这个链接也不是bert的。

**Pull request merged**